src/ImageRunModal.jsx

```diff
@@ -407,7 +404,7 @@ export class ImageRunModal extends React.Component {
 
         // If there are registries configured search in them, or if a user searches for `docker.io/cockpit` let
         // podman search in the user specified registry.
-        if (Object.keys(this.props.registries).length !== 0 || value.includes('/')) {
+        if (Object.keys(this.props.podmanInfo.registries).length !== 0 || value.includes('/')) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1249-20230710-125104-d428b4fe-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)